index.rst

```diff
@@ -0,0 +1,15 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO Developers documentation!
+=====================================
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
We should be careful with this setting to not overload menu. I would start with it set to 1.

Reply:
I will leave it for now, will check if this will overload menu as we will enter more documentations